src/reset.css

```diff
@@ -5,47 +5,42 @@
   box-sizing: border-box;
 }
 
-/* Remove default padding */
-ul[class],
-ol[class] {
-  padding: 0;
-}
-
 /* Remove default margin */
 body,
 h1,
 h2,
 h3,
 h4,
 p,
-ul[class],
-ol[class],
 figure,
 blockquote,
 dl,
 dd {
   margin: 0;
 }
 
+/* Remove list styles on ul, ol elements with a list role, which suggests default styling will be removed */
+ul[role="list"],
+ol[role="list"] {
+  list-style: none;
+}
+
 /* Set core root defaults */
 html {
+  min-height: 100vh;
+  min-height: -webkit-fill-available;
   scroll-behavior: smooth;
-  height: -webkit-fill-available;
 }
 
 /* Set core body defaults */
 body {
   min-height: 100vh;
-  /* mobile viewport bug fix */
   min-height: -webkit-fill-available;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
```

Comment:
I can already tell you, as a Linux user, this is one of the most annoying trends. This statically overwrites the browser CSS settings, which allows setting serif and sans-serif standards. But with this line of code, you remove the possibility, enforcing, at least for me on Linux, Ubuntu font on all websites, which is absolutely not really convenient. The sensible default, from my point of view, is using `serif` or `sans-serif`.

Reply:
That's my bad, embarrassingly enough I actually meant to add those more opinionated styles to a personal fork of mine, not here. This was just meant to be the `-webkit-fill-available` fix. All the same, that's a good point, I'm not a Linux user, so I'll take that into consideration.